test/functional/editor_controller_test.rb

```diff
@@ -69,7 +77,7 @@ def setup
 
   test "should show title form input if title parameter present" do
```

Comment:
So, for each test which you've switched to test the "legacy" action, we should add the word "legacy" to the description. But second, we should really create new tests (they could be almost copies of the originals) to test the new template in each of the ways we've tested the legacy template. That way, both are tested and we are more assured that they're not broken as the code advances. Make sense? Thank you for your patience, we are very close!